java/code/src/com/redhat/rhn/frontend/action/systems/sdc/SystemOverviewAction.java

```diff
@@ -65,8 +67,19 @@ public class SystemOverviewAction extends RhnAction {
     public ActionForward execute(ActionMapping mapping, ActionForm form,
             HttpServletRequest request, HttpServletResponse response) {
         RequestContext rctx = new RequestContext(request);
-        Long sid = rctx.getRequiredParam("sid");
         User user = rctx.getCurrentUser();
+        Long sid = null;
+        String name = rctx.getParam("name", false);
+        if (name != null) {
+            List<SystemOverview> matches = SystemManager.listSystemsByName(user, name);
+            if (matches.size() != 1) {
+                throw new BadParameterException("No or multiple systems matching the name");
+            }
```

Comment:
![image](https://user-images.githubusercontent.com/2827771/206223763-fb9e9124-a6bc-4f01-8cde-3481a6f36653.png)
Good to catch this :)